# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../isdn4k-utils/empty-switch-label.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- isdn4k-utils/lib/libtools.c~	1999-11-03 17:13:36.000000000 +0100
+++ isdn4k-utils/lib/libtools.c	2004-10-14 14:02:56.732530824 +0200
@@ -710,8 +710,8 @@
 				          break;
 				case 'r': _mode |= R_OK;
 				          break;
-				default :
+				/*default :*/
 			}
 		}
 

--- isdn4k-utils/isdnlog/tools/rate.c.orig	2002-07-06 01:22:33.000000000 +0200
+++ isdn4k-utils/isdnlog/tools/rate.c	2004-10-14 14:18:52.669206360 +0200
@@ -1772,8 +1772,8 @@
 	  numbers++;
 	  number=realloc(number, numbers*sizeof(int));
 	  number[numbers-1]=i;
-	skip:
 	}
+      skip:
 
 	while (isblank(*s)) s++;
 	if (*s==',') {
@@ -2343,10 +2343,10 @@
 	  }
 	}
 	return UNKNOWN;
-      done:
       }
     }
   }
+  done:
 
   if (Rate->_zone<0 || Rate->_zone>=Provider[prefix].nZone) {
     return UNKNOWN;
